app/src/main/res/values/strings.xml

```diff
@@ -104,4 +104,6 @@
     <string name="selfie_instruction">to take a selfie to continue your registration.</string>
     <string name="recapture_tutorial">Click on the button to try capturing the tree again.</string>
     <string name="capture_tutorial">Click on the button to save tree.</string>
+    <string name="no_gps_device_header">NO GPS DEVICE FOUND</string>
+    <string name="no_gps_device_content">This app is not supported on this device because no GPS receiver was found. Kindly try another phone that has GPS device</string>
```

Comment:
I think we can reword this to say something more like 
"This device does not support GPS. Please use a device that has GPS to continue."

Reply:
We also need a workflow for making sure we have a swahili translation for any new text added.  @sebastiangaertner can provide this.  Please file a ticket for him.